content/resources/user/submit-data/sower.md

```diff
@@ -0,0 +1,119 @@
+    ```
+    $/usr/local/bin: aws configure
+    AWS Access Key ID [*******************2]:
+    AWS Secret Access Key [*******************H]:
+    Default region name [None]: us-east-1
+    Default output format [None]:
+    ```
+
+- To access the bucket structure and find files, run:
+    ```
+    aws s3 ls s3//:bucketname
+    ```
+
+- To copy files to your local environment run `aws s3 cp s3://1000genomes/[path-to-file] [local-path]`:
+    ```
+    aws s3 cp s3://1000genomes/sequence_indices/20091216_20100311.stats.csv /usr/local/bin/sower_test/
+    ```
+    More documentation about AWS CLI terminal commands can be found [here](https://aws.amazon.com/cli/#file_commands_anchor).
+
+- After finding the path to files of interest in the bucket, calculating their size and md5, the manifest can be created. In our example, the access is open, so an asterisk can be inserted into the `acl` column:
+
+    ```
+    GUID	url	size	md5	acl
+     	s3://1000genomes/sequence_indices/20091216_20100311.stats.csv	745	613ccfbc68287db60c663519dab6a4ff	['*']
+    ```
+
+    This file will be now indexed by clicking "Index Files" in the [UI](https://gen3.datacommons.io/indexing).
+
+- The resulting manifest including the GUID can be downloaded as `.log` after clicking "Download Manifest":
+    ```
+    guid	urls	size	md5	acl
+    c146a966-7617-4f2f-b628-57a5f11de879	s3://1000genomes/sequence_indices/20091216_20100311.stats.csv	745	613ccfbc68287db60c663519dab6a4ff	['*']
+    ```
+
+- The submission files for the metadata can be now prepared following the instructions described in the [data submission](https://gen3.org/resources/user/submit-data/#4-submit-additional-project-metadata) or briefly described below. For this example we will use Windmill for the submission and show the steps as described as option 1 above, since it demonstrates the ability to link to a file node that is deep within the graph model.
+
+- For the quick metadata file upload that links to the core_metadata_collection, download the template.tsv for the `core_metadata_collection` node in the [data dictionary](https://gen3.datacommons.io/DD). Fill in the required (*) columns, according to the data dictionary. The core_metadata_collection.tsv should contain `type`, `submitter_id`, and `projects.code`:
+
+    ```
+    type	submitter_id	projects.code
+    core_metadata_collection	xritterdemo1testsower	test
+    ```
+    Submit and upload this file. A green message should appear after successful upload:
+    `Succeeded: 200
+    Submitted chunk 1 of 1`
+
+    >__NOTE:__  the `id` is created on the backend. To find it, download the core_metadata_collection.tsv under Submit Data --> Submit Data to your project --> Core Metadata Collection --> Download All as TSV.
+
+- Next, download the template.tsv for the `data_file` node of your choice in the [data dictionary](https://gen3.datacommons.io/DD), colored in green. In this example, we use the `Variant Cell File (VDF)` data_file node.
```

Comment:
I think you meant Variant Call File (VCF).

Reply:
done